src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+    {
+    Question: "What is the best algorithm for matrix multiplication?",
+    Answer: "Strassen's Algorithm",
+    Distractor1: "Multiplication algorithm",
+    Distractor2: "Divide and Conquer Technique",
+    Distractor3: "None of the above",
+    Explanation: "Strassen's reduces number of mumtiplication and addition in givven matrix and make time complexity O(2.81)",
+    Link: "https://forum.freecodecamp.org/t/matrix-multiplication-strassen-algortihm-recursive/557115",
```

Comment:
You should only use forum links if they are posted by Camperbot because other forum links can be deleted by the user at any moment. 

```suggestion
    Link: "https://en.wikipedia.org/wiki/Strassen_algorithm#",
```